### ISSUE TITLE
Fix GHA builds

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/activeuuid#61]
@skalee could you please help fix the GHA builds?

And update the GHA config with the previous Travis matrix. Maybe we should combine the workflow files for the same matrix... Up to you. Thanks!

```yaml
-  - jruby
-
-gemfile:
-  - gemfiles/Rails-5_2.gemfile
-
-env:
-  - DB=sqlite3
-  - DB=mysql
-  - DB=postgresql
-
-matrix:
-  include:
-    - gemfile: gemfiles/Rails-head.gemfile
-      env: DB=sqlite3
-    - gemfile: gemfiles/Rails-head.gemfile
-      env: DB=mysql
-    - gemfile: gemfiles/Rails-head.gemfile
-      env: DB=postgresql
-    - gemfile: gemfiles/Rails-5_1.gemfile
-      env: DB=sqlite3
-    - gemfile: gemfiles/Rails-5_1.gemfile
-      env: DB=mysql
-    - gemfile: gemfiles/Rails-5_1.gemfile
-      env: DB=postgresql
-    - gemfile: gemfiles/Rails-5_0.gemfile
-      env: DB=sqlite3
-    - gemfile: gemfiles/Rails-5_0.gemfile
-      env: DB=mysql
-    - gemfile: gemfiles/Rails-5_0.gemfile
-      env: DB=postgresql
-    # Without monkey patches
-    - env: DB=sqlite3 NO_PATCHES=1
-    - env: DB=mysql NO_PATCHES=1
-    - env: DB=postgresql NO_PATCHES=1
-  allow_failures:
-    - gemfile: gemfiles/Rails-head.gemfile
-    - rvm: jruby
```

_Originally posted by @ronaldtse in https://github.com/riboseinc/activeid/pull/60#issuecomment-636425614_

**Comment by skalee** [comment]
@ronaldtse from the workflow YAMLs:

```
# Auto-generated by Cimas: Do not edit it manually!
# See https://github.com/metanorma/cimas
```

I guess this is some copy-paste leftover, isn't it? Or maybe this repository is meant to be handled by Cimas?

**Comment by ronaldtse** [comment]
@skalee copy-paste leftover...

**Comment by skalee** [comment]
Fixed in #62.
